File: program/print-elements-of-an-array/PrintElementsOfAnArray.c
# include <stdio.h>

// Prints int array
void PrintArray(int *array)
{
	int len = sizeof(array);
	for (int i = 0; i < len; ++i)
	{
		printf("%d\n", array[i]);
	}
}

void main()
{
	int array[] = {1, 2, 3, 4};
	PrintArray(array);
}

File: program/print-elements-of-an-array/printElementsOfAnArray.scala
object printArray {
    def main(args:Array[String]):Unit={
        val nums: List[Int] = List(1, 2, 3, 4, 5); 
        for ( i <- 0 to (nums.length - 1)) {
         print(nums{i});
        }
    }
}
